src/sentry/static/sentry/app/views/alerts/details/header.tsx

```diff
@@ -172,11 +203,12 @@ const StyledLoadingError = styled(LoadingError)`
   }
 `;
 
-const GroupedHeaderItems = styled('div')`
+const GroupedHeaderItems = styled('div')<{columns: number}>`
```

Comment:
React may yell at you here since emotion will pass this prop to the el it is styling (the `div`) -- if you make it `camelCased`, emotion will not (there are also other ways (grep for `isPropValid`) but I think this will be cleanest).